config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   resources :books, only: [:index]
+  resources :authors, only: [:show]
+  get "/authors/:id/books", to: "authors#books"
```

Comment:
This non-RESTful route might be uneccessary. For an MVP, and following the specs on Upcase, I think it's fine to just render up to 5 books on the Author `show` page (perhaps in a book partial, after initially implemented without a partial).